library/psa_crypto.c

```diff
@@ -2063,8 +2063,10 @@ static const mbedtls_md_info_t *mbedtls_md_info_from_psa( psa_algorithm_t alg )
             return( &mbedtls_sha256_info );
 #endif
 #if defined(MBEDTLS_SHA512_C)
+#if !defined(MBEDTLS_SHA512_NO_SHA384)
         case PSA_ALG_SHA_384:
```

Comment:
Missing: `case PSA_ALG_SHA_384` in `psa_hash_setup` (the other cases have negligible impact)